generators/emails/templates/example_email.rb

```diff
@@ -3,6 +3,6 @@ class TestEmail < Howitzer::Email
   subject 'Test email'
 
   def addressed_to?(new_user)
-    /Hi #{new_user}/ === plain_text_body
+    plain_text_body =~ /Hi #{new_user}/
```

Comment:
it will return number if matches, and predicate methods should return true or false